bbc/simorgh
# Consent container banner

Resolves #1399

**Overall change:** Creates container for cookie and privacy banner

**Code changes:**

- Creates container for cookie and privacy banner
- Adds `/ckns_policy/*` dev route to allow endpoint to use locally.

**Testing:**
1) Check the banner follows the logic described in the description of  https://github.com/bbc/simorgh/issues/1399
2) Check the banner works as expected across devices

Note this will only work on canonical pages e.g.http://localhost:7080/news/articles/cn7769kpk9mo,  not amp pages

Message Drew if you have any questions :)

---
